lib/elastic_record/search_hit.rb

```diff
@@ -0,0 +1,48 @@
+module ElasticRecord
+  module SearchHit
+    def instantiate_from_hit(hit)
```

Comment:
```suggestion
    def from_search_hit(hit)
```

Reply:
Follow the same pattern used with `AsDocument` and inject the mapping
https://github.com/data-axle/elastic_record/blob/8cb048d5fc96d42ec3bdc2766e050a0523e09df0/lib/elastic_record/as_document.rb#L3-L11